### PR TITLE
Port hastebin to Sir Robin

### PR DESCRIPTION
**Pull request by Bluenix2**
Formatted code can become longer than its input, this means that the message sent may go over the 2000-character limit.

This PR ports over the bot service file and puts it to good use in the blurple formatter.

### CODE REVIEW

File: bot/exts/blurple_formatter.py [hunk 1]
```diff
@@ -2,11 +2,14 @@
...
 import discord
+from botcore.utils.logging import get_logger
 from botcore.utils.regex import FORMATTED_CODE_REGEX
 from discord.ext import commands
 
 from bot.bot import SirRobin
-from bot.utils import blurple_formatter
+from bot.utils import blurple_formatter, services
+
+log = get_logger(__name__)
```
Comment (wookie184) [inline bot/exts/blurple_formatter.py:12]: This doesn't seem to be used anywhere

**Review by wookie184 — CHANGES_REQUESTED** [review]
One thing

File: bot/exts/blurple_formatter.py (hunk 1 above)
Comment (Bluenix2) [reply bot/exts/blurple_formatter.py]: Ah, yes you're are right. This was left from another attempt I made - oops. Fixed in e345ae5d64afbf7f93a6dc637665009c9451568d

**Review by wookie184 — APPROVED** [review]
LGTM

**Review by D0rs4n — APPROVED** [review]
Works like a charm!

**Pull request merged**